utils/general.py

```diff
@@ -36,6 +38,73 @@
 # Prevent OpenCV from multithreading (to use PyTorch DataLoader)
 cv2.setNumThreads(0)
 
+# Download objects
+def download_file(dir, blob):
+    file_name = blob.name.split("/")[2]
+    destination = Path(dir) / f"{file_name}"
+    destination.touch()
+    blob.download_to_filename(destination)
+
+
+# Place GCS objects in their respective local directories
+def place_object(local_dir, blob):
+    image_train = "images/t"
+    image_val = "images/v"
+    label_train = "labels/t"
+    label_val = "labels/v"
+    jpg = ".jpg"
+    txt = ".txt"
```

Comment:
For simplicity, you can omit the variables, in this case. `jpg` and `"jpg"` is sufficiently close that you can just insert the values directly.

Reply:
Fixed a33490a6a6b0d8860a9e67280bf0972a23f022c3